app/Http/Controllers/BookController.php

```diff
@@ -17,10 +17,19 @@ public function __construct(Book $book)
         $this->book=$book;
     }
 
-    public function getAllBooks()
+    public function getAllBooks(Request $request)
     {
         $books = $this->book->all();
-
+        
+        if (isset($request->claimed)) {
```

Comment:
You actually don't need to use isset here - if the request doesn't have claimed, it will be automatically set to null.

You can just do `if ($request->claimed)` to get the same outcome

Reply:
Fixed!